resources/definitions/flashforge_dreamer_nx.def.json

```
@@ -0,0 +1,31 @@
+{
+    "version": 2,
+    "name": "Dreamer NX",
+    "inherits": "flashforge_base",
+    "metadata": {
+        "visible": true,
+        "author": "Egon",
+        "manufacturer": "Flashforge",
+        "file_formats": "text/x-gcode",
+        "platform": "FlashForge_DreamerNX.obj",
+        "platform_offset": [0, 0, 0],
+        "quality_definition": "flashforge_base",
```

I think this comma is the cause of the failing tests.
```suggestion
        "quality_definition": "flashforge_base"
```